superset/security/manager.py

```diff
@@ -177,15 +177,13 @@ def can_access(self, permission_name: str, view_name: str) -> bool:
             return self.is_item_public(permission_name, view_name)
         return self._has_view_access(user, permission_name, view_name)
 
-    def can_only_access_owned_queries(self) -> bool:
+    def can_access_all_queries(self) -> bool:
         """
-        Return True if the user can only access owned queries, False otherwise.
+        Return True if the user can access all queries, False otherwise.
 
-        :returns: Whether the use can only access owned queries
+        :returns: Whether the use can access all queries
         """
-        return self.can_access(
-            "can_only_access_owned_queries", "can_only_access_owned_queries"
-        )
+        return self.can_access("can_access_all_queries", "can_access_all_queries")
```

Comment:
The naming of this permission is inconsistent with the naming of other similar permissions. If we wished to bring this in line with `all_datasource_access` and `all_database_access` I would recommend altering this name to `all_query_access`. `can_access_` in the permission name is redundant.

Reply:
changed